### ISSUE TITLE
[csolution] no IAR version in cprj ?

### ISSUE THREAD

**Issue by fred-r** [Open-CMSIS-Pack/devtools#217]
yml file:
```
  build-types:
    - type: AC6
      compiler: AC6
      misc:
        - C*:
          - -std=c99
    - type: GCC
      compiler: GCC
      misc:
        - C*:
          - -std=c99
    - type: IAR
      compiler: IAR
      misc:
        - C*:
          - -std=c99          
```

cprj file:
```
  <compilers>
    <compiler name="IAR" version="0.0.0"/>
  </compilers>
```

==> I would have expected version 8.50.6 as we have IAR.8.50.6.cmake

**Comment by jkrech** [comment]
@fred-r could you please validate that this problem got fixed in csolution 1.1.0.

**Comment by ReinhardKeil** [comment]
@JonatanAntoni could you create an example that shows switching from AC6, GCC, IAR using a cdefault.yml file?
this releates to #615

**Comment by JonatanAntoni** [comment]
We have an example that shall work for all three compilers here: https://github.com/ARM-software/CMSIS-View/tree/eventstatistic/Examples/EventStatistic

The example is still work in progress and doesn't compile for IAR as it depends on other open PRs still in progress as well. *But* it can be used to verify if the generated `.cprj` is as expected for IAR compiler:

```bash
./CMSIS-View/Examples/EventStatistic $ cp EventStatistic.iar-cdefault.yaml EventStatistic.cdefault.yaml

./CMSIS-View/Examples/EventStatistic $ csolution convert -s EventStatistic.csolution.yaml -c EventStatistic.Debug+CM3
./EventStatistic.Debug+CM3-IAR/EventStatistic.Debug+CM3.cprj - info csolution: file generated successfully
./EventStatistic.cbuild-idx.yml - info csolution: file generated successfully
./EventStatistic.Debug+CM3-IAR/EventStatistic.Debug+CM3.cbuild.yml - info csolution: file generated successfully

./CMSIS-View/Examples/EventStatistic $ grep 'compiler name="IAR"' EventStatistic.Debug+CM3-
IAR/EventStatistic.Debug+CM3.cprj
    <compiler name="IAR" version="9.30.1"/>
```

The problem seems to be fixed in latest csolution 1.4.0.
